Week1/hm/queries4.js

```diff
@@ -0,0 +1,39 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password : 'hyfpassword',
+    database: 'world'
+});
+
+connection.connect();
+
+const queries =[
+    // countries with population greater than 8 million
+    'SELECT name FROM country WHERE population>=8000000',
+
+    // countries that have “land” in their names
+    'SELECT name FROM country WHERE name LIKE "%lands"',
+
+    // cities with population in between 500,000 and 1 million
+    'SELECT name FROM city WHERE population>=500000 and population<= 1000000',
```

Comment:
Another option would have been to use between. More info here: https://www.techonthenet.com/mysql/between.php